manifests/init.pp

```diff
@@ -120,6 +136,11 @@
     amqp_truststore              => $::certs::candlepin::amqp_truststore,
     qpid_ssl_cert                => $::certs::qpid::client_cert,
     qpid_ssl_key                 => $::certs::qpid::client_key,
+    db_host                      => $katello::candlepin_db_host,
```

Comment:
@mbacovsky I don't think that's worth it. They're static in candlepin, except the password but that's handled by using the same password cache